# Data-Plus-2023

Repository for Duke Data+ 2023 Team 11

## Team Members

### Team Lead

Dr. Amy Schmid

### Project Manager

Christopher Kilner

### Team Members

Aiden Pasinsky

Anthony Du

## Project Description

The goal of this Data+ project is to apply and extend custom analytics solutions to understand and predict microbial population growth. An explosion of data has resulted from tracking the growth of bacteria in high throughput devices. These data were generated to understand how microbes grow. Better models that fit and predict these growth data are needed for better treatment of pathogenic bacterial infections, food safety, beer and bread fermentation, and understanding stress resilience of the microbiome. Using nonparametric statistical models to analyze how microbes grow under stress, the Schmid research lab at Duke has made important discoveries in these areas. These studies generated large data sets and developed statistical models to track and predict how microbes grow and change their gene expression when faced with extreme stress. We built a web application called phenom to make these models accessible to the broader community. In this Data+ project, students will beta test the web app and make improvements, including data visualization, extending the underlying statistical model, and analyzing data using the app.

## Background Papers

| Paper                                                                                                                                                           | Purpose          | Lab        | Notes                                                                                                                              |
|----------------|------------------|------------|---------------------------|
| [Sharma, et. al.](https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-13-351)                                                                      | Data+ and WebApp | Schmid     | Data and metadata located in supplementary table                                                                                   |
| [Tonner, et. al.](https://genome.cshlp.org/content/early/2017/01/23/gr.210286.116)                                                                              | Data+ and WebApp | Schmid     | Data and metadata located in paper github repo (example dataset for running B-GREAT model)                                         |
| [Bushell, et. al.](https://www.frontiersin.org/articles/10.3389/fmicb.2018.03196/full)                                                                          | Data+ and WebApp | Lund       | Data and metadata located in supplementary tables. good test case for models applied to data from a different growth curve machine |
| [Midani, et. al.](https://journals.asm.org/doi/full/10.1128/mSystems.00508-21)                                                                                  | WebApp           | Britton    | Potential alternative model to B-GREAT                                                                                             |
| [Zwietering, et. al.](https://journals.asm.org/doi/10.1128/aem.56.6.1875-1881.1990?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed) | Data+            | Zwietering | Equations for conducting regression on microbial growth data                                                                       |
|                                                                                                                                                                 |                  |            |                                                                                                                                    |

: All team members should familiarize themselves with these papers

## Weekly Goals and Agenda
### Week 1
Aiden: 
Set up GitHub and RStudio on computer - Completed
Read and analyze papers - Completed 
  Question - I am unsure why the AMiGA paper is an alternative model to B-Great.
  After reading the paper it seems AMiGA still uses the GP regression models 
  proposed in the Tonner paper but is just an easily accessible system that 
  allows people inexperienced with programming to use.
Test and experiment with ggplot themes - Completed
### Week 2

### Week 3

### Week 4

### Week 5

### Week 6

### Week 7

### Week 8

### Week 9

### Week 10